bccsp/schemes/aries/signer.go

```diff
@@ -50,7 +54,15 @@ func (s *Signer) getPoKOfSignature(
 
 	messagesFr := credential.toSignatureMessage(sk, s.Curve)
 
-	pokOS, err := bbs12381g2pub.NewBBSLib(s.Curve).NewPoKOfSignature(signature, messagesFr, revealedAttributesIndex(attributes), ipk)
+	var pokOS *bbs12381g2pub.PoKOfSignature
+	if sigtype == types.Smartcard {
```

Comment:
could you add a comment to explain the difference between the two cases?

Reply:
done